mode/examples/Topics/Motion/CubesWithinCube/CubesWithinCube.pyde

```diff
@@ -6,22 +6,21 @@ Collision detection against all outer cube's surfaces.
 """
 from cube import Cube
 
-halfWidth = None
-halfHeight = None
 
 Bounds = 300
 # 20 little internal cubes
 cubes = []
-for i in range(20):
```

Comment:
The only reason I can think of for moving this sort of initialization into setup() is needing to know the ```width``` or ```height``` or something. What's wrong with leaving it here?